packages/flutter/test/material/toggle_buttons_test.dart

```diff
@@ -1398,6 +1398,155 @@ void main() {
     },
   );
 
+  testWidgets(
+    'Properly draws borders based on state when direction is vertical and verticalDirection is down.',
+        (WidgetTester tester) async {
+      final ThemeData theme = ThemeData();
+      await tester.pumpWidget(
+        Material(
+          child: boilerplate(
+            child: ToggleButtons(
+              direction: Axis.vertical,
+              verticalDirection: VerticalDirection.down,
+              isSelected: const <bool>[false, true, false],
+              onPressed: (int index) {},
+              children: const <Widget>[
+                Text('First child'),
+                Text('Second child'),
+                Text('Third child'),
+              ],
+            ),
+          ),
+        ),
+      );
+
+      // The children should be laid out along vertical and the first child at top.
```

Comment:
Might be worth adding a short comment explaining the 1.0 pixel differences (due to the border drawings)